glue/viewers/matplotlib/state.py

```diff
@@ -34,13 +37,69 @@ def notify(self, *args, **kwargs):
 
 VALID_WEIGHTS = ['light', 'normal', 'medium', 'semibold', 'bold', 'heavy', 'black']
 
-VALID_LOCATIONS = ['best',
+VALID_LOCATIONS = ['best (draggable)', 'best',
```

Comment:
What about just ``'draggable'`` for the first option?

Reply:
Much better indeed !